kuma/wiki/models.py

```diff
@@ -1735,10 +1749,6 @@ def get_tidied_content(self, allow_none=False):
         self.tidied_content = tidied_content or ''
         return tidied_content
 
-    @property
-    def content_cleaned(self):
```

Comment:
I think we'll need this for historical revisions. Once this ships, we can try to distinguish between new revisions and unsafe old revisions, using a revision ID, a new boolean, or other method.